### PR TITLE
Standardize language in docs files

### PR DESCRIPTION
**Pull request by kentdanas**
# Description
Update documentation to use standardized language in line with Astronomer's style guide.

### CODE REVIEW

File: docs/astro/sql/operators/load_file.rst [hunk 1]
```diff
@@ -113,43 +113,42 @@ Parameters to use when loading a file to pandas dataframe
...
+        Check for valid parameters based on **file location** and **database** combination in :ref:`supported_native_path`
 
         .. literalinclude:: ../../../../example_dags/example_load_file.py
            :language: python
            :start-after: [START load_file_example_8]
            :end-before: [END load_file_example_8]
 
 
-#. **enable_native_fallback** -  When ``use_native_support`` is True, we try to use the native transfer, and if this fails we try to use the default path to load data, giving the user a warning. If you want to change this behavior pass ``enable_native_fallback=False``.
+#. **enable_native_fallback**: When ``use_native_support`` is set to True, ``load_file`` will attempt to use native transfer. If this fails, ``load_file`` will attempt to use the default path to load data and you will see a warning. If you want to change this behavior you can specify ``enable_native_fallback=False``.
```
Comment (feluelle) [inline docs/astro/sql/operators/load_file.rst:151]:
```suggestion
#. **enable_native_fallback**: When ``use_native_support`` is set to ``True``, ``load_file`` will attempt to use native transfer. If this fails, ``load_file`` will attempt to use the default path to load data and you will see a warning. If you want to change this behavior you can specify ``enable_native_fallback=False``.
```

File: docs/astro/sql/operators/transform.rst [hunk 2]
```diff
@@ -4,20 +4,18 @@
 transform operator
 ==================
 
-
 When to use the ``transform`` operator
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The transform function of the SQL decorator is the ``T`` of the ELT system. Each step of the transform pipeline creates a new table from the ``SELECT`` statement and enables tasks to pass those tables as if they were native Python objects. The transform: applies a SQL select statement to a source table and saves the result to a destination table.
+The ``transform`` operator allows you to implement the ``T`` of an ELT system by running a SQL query. Each step of the transform pipeline creates a new table from the ``SELECT`` statement and enables tasks to pass those tables as if they were native Python objects.
```
Comment (feluelle) [inline docs/astro/sql/operators/transform.rst:9]:
```suggestion
The ``transform`` operator allows you to implement the **T** of an ELT system by running a SQL query. Each step of the transform pipeline creates a new table from the ``SELECT`` statement and enables tasks to pass those tables as if they were native Python objects.
```

File: docs/astro/sql/operators/transform.rst [hunk 3]
```diff
@@ -4,20 +4,18 @@
...
-There are two use cases of the ``transform`` operator.
+There are two main uses for the ``transform`` operator.
 
-Case 1: When we pass tables between tasks while completing a data transformation.
+Case 1: Passing tables between tasks while completing data transformations.
     The following example applies a SQL ``SELECT`` statement to a ``imdb_movies`` table with templating and saves the result to a ``top_animation`` table.
 
-    Please note ``input_table`` in the double brackets is treated as Airflow jinja templates. This is NOT an f string. F-strings in SQL formatting risk security breaches via SQL injections. For security, users MUST explicitly identify tables in the function parameters by typing a value as a Table. Only then will the SQL decorator treat the value as a table.
-
+    Note that the ``input_table`` in the double brackets is treated as an Airflow jinja template. It is NOT an f string. F-strings in SQL formatting are at risk of security breaches via SQL injections. For security, you MUST explicitly identify tables in the function parameters by typing a value as a Table. Only then will the ``transform`` operator treat the value as a table.
```
Comment (feluelle) [inline docs/astro/sql/operators/transform.rst:18]:
```suggestion
    Note that the ``input_table`` in the double brackets is treated as an Airflow jinja template. It is **not** an f string. F-strings in SQL formatting are at risk of security breaches via SQL injections. For security, you **must** explicitly identify tables in the function parameters by typing a value as a Table. Only then will the ``transform`` operator treat the value as a table.
```

**Review by feluelle — APPROVED** [review]
Thank you very much. It looks so much better. I have a few suggestions though (see below). Let me know what you think.

File: docs/astro/sql/operators/dataframe.rst [hunk 4]
```diff
@@ -7,18 +7,18 @@ dataframe operator
...
+The ``dataframe`` operator allows you to run Python transformations in Airflow. Behind the scenes, the ``dataframe`` function automatically coverts the source SQL table into a Pandas dataframe, and makes any dataframes resulting from the transformation available to downstream astro.sql functions. This means you can seamlessly transition between Python and SQL for data transformations without writing any code to explicitly do so. To use the ``dataframe`` operator, you simply provide a Python function that takes a dataframe as one of its inputs, and specify a ``Table`` object as the input SQL table. If you want the resulting dataframe to be converted back to SQL, you can specify an ``output_table`` object.
```
Comment (sunank200) [inline docs/astro/sql/operators/dataframe.rst:10]:
```suggestion
The ``dataframe`` operator allows you to run Python transformations in Airflow. Behind the scenes, the ``dataframe`` function automatically coverts the source SQL table into a Pandas dataframe, and makes any dataframes resulting from the transformation available to downstream ``astro.sql`` functions. This means you can seamlessly transition between Python and SQL for data transformations without writing any code to explicitly do so. To use the ``dataframe`` operator, you simply provide a Python function that takes a dataframe as one of its inputs, and specify a ``Table`` object as the input SQL table. If you want the resulting dataframe to be converted back to SQL, you can specify an ``output_table`` object.
```

**Review by sunank200 — APPROVED** [review]
Thank you very much for the changes. It looks much better now. I have added few suggestions.

**Review by kaxil — APPROVED** [review]
Great work @kentdanas , this looks great

**Pull request merged**